client/server.js

```diff
@@ -30,7 +30,7 @@ server.app.post('/comments.json', function(req, res) {
   console.log('Just got done with nap!');
   comments.push(req.body.comment);
   res.setHeader('Content-Type', 'application/json');
-  res.send(JSON.stringify(req.body.comment));
+  res.send(JSON.stringify(comments));
```

Comment:
send back all comments? OK I guess if we knew the list was limited.
I have it coded to send back just the comment that's saved.